scripts/updateConsent.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+const {Pool} = require('pg');
+
+function updateConsent(database){
+  //Still need to grab emails from a sensitive data file
+  const emails = JSON.parse(fs.readFileSync('./tmp/test.json'))
+  const emailString = emails.consented.map(x => "'" + x + "'").toString();
+  const pool = new Pool({database});
+  const sql = `
+    INSERT INTO consented_email (email,audio,permission,consent) 
+    SELECT x,TRUE,TRUE,TRUE 
+    FROM unnest(ARRAY[${emailString}]) x;`;
```

Comment:
⚠️ Making SQL commands out of strings allows a class of security vulnerabilities called "SQL injection" and is really dangerous :)

![image](https://user-images.githubusercontent.com/1056957/38222059-c0968b56-36b0-11e8-9b93-fa7c3ed99bee.png)

So here you can imagine if someone filled out our consent form with some crazy kind of email address, they might be able to get this string to be something that we wouldn't want.

If you want to input with a SQL statement, you should never form the SQL string manually, but instead use either "escaping" functions that the library provides, or a method that lets you pass in arguments separately.  Here's an example from elsewhere in this codebase that you can use as a model: https://github.com/mit-teaching-systems-lab/threeflows/blob/1696e7f01886121d9154123dfcd6819bfd051747/server/migrations/20160718-001-insert-evaluations.js#L38

Reply:
Haha. I've seen that comic before!

This is my bad. I've been following a template for sql queries and this slipped my mind while I hacked this bit together. I've pushed a fix!